src/styles/page.scss

```
@@ -0,0 +1,75 @@
+.page {
+  font-family: "Poppins", sans-serif;
+  font-size: 16px;
+  line-height: 140%;
+  color: $colorWhite;
+
+  background-color: $colorDark;
+
+  scroll-behavior: smooth;
+
+  &__body {
+    margin: 0;
+
+    &--with-menu {
+      overflow: hidden;
+    }
+  }
+
+  &__menu {
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
+
+    transform: translateX(100%);
+    transition: transform 0.3s;
```

Replace in all cases
```suggestion
    transition: transform $efectsDuration;
```